packages/tables/backend/sirius-components-tables/src/main/java/org/eclipse/sirius/components/tables/descriptions/ColumnDescription.java

```diff
@@ -0,0 +1,135 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.tables.descriptions;
+
+import java.text.MessageFormat;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+import java.util.function.Function;
+
+import org.eclipse.sirius.components.annotations.Immutable;
+import org.eclipse.sirius.components.representations.VariableManager;
+
+/**
+ * The description of the column.
+ *
+ * @author arichard
+ * @author lfasani
+ */
+@Immutable
+public final class ColumnDescription {
+
+    public static final String COLUMN_TARGET_OBJECT = "columTargetObject";
+
+    public static final String COLUMN_TARGET_OBJECT_ID = "columTargetObject";
```

Comment:
Why does this contant has the same value as the one just above? Is the constant necessary, one could ask for the id of the column target object.